includes/definitions/discovery/apc.yaml

```diff
@@ -2,34 +2,294 @@ mib: PowerNet-MIB
-                        - { descr: ecoMode, graph: 0, value: 13, generic: 0 }
-                        - { descr: hotStandby, graph: 0, value: 14, generic: 0 }
-                        - { descr: onBatteryTest, graph: 0, value: 15, generic: 1 }
-                        - { descr: emergencyStaticBypass, graph: 0, value: 16, generic: 2 }
-                        - { descr: staticBypassStandby, graph: 0, value: 17, generic: 1 }
-                        - { descr: powerSavingMode, graph: 0, value: 18, generic: 0 }
-                        - { descr: spotMode, graph: 0, value: 19, generic: 0 }
-                        - { descr: eConversion, graph: 0, value: 20, generic: 0 }
-                        - { descr: chargerSpotmode, graph: 0, value: 21, generic: 0 }
-                        - { descr: inverterSpotmode, graph: 0, value: 22, generic: 0 }
\ No newline at end of file
+            -
+                oid: upsBasicOutputStatus
+                num_oid: .1.3.6.1.4.1.318.1.1.1.4.1.1.
+                descr: Output Status
+                index: '{{ $index }}'
+                states:
+                    - { descr: unknown, graph: 0, value: 1, generic: 3 }
+                    - { descr: onLine, graph: 0, value: 2, generic: 0 }
+                    - { descr: onBattery, graph: 0, value: 3, generic: 1 }
+                    - { descr: onSmartBoost, graph: 0, value: 4, generic: 0 }
+                    - { descr: timedSleeping, graph: 0, value: 5, generic: 0 }
+                    - { descr: softwareBypass, graph: 0, value: 6, generic: 1 }
+                    - { descr: off, graph: 0, value: 7, generic: 2 }
+                    - { descr: rebooting, graph: 0, value: 8, generic: 1 }
+                    - { descr: switchedBypass, graph: 0, value: 9, generic: 1 }
+                    - { descr: hardwareFailureBypass, graph: 0, value: 10, generic: 2 }
+                    - { descr: sleepingUntilPowerReturn, graph: 0, value: 11, generic: 1 }
+                    - { descr: onSmartTrim, graph: 0, value: 12, generic: 1 }
+                    - { descr: ecoMode, graph: 0, value: 13, generic: 0 }
+                    - { descr: hotStandby, graph: 0, value: 14, generic: 0 }
+                    - { descr: onBatteryTest, graph: 0, value: 15, generic: 1 }
+                    - { descr: emergencyStaticBypass, graph: 0, value: 16, generic: 2 }
+                    - { descr: staticBypassStandby, graph: 0, value: 17, generic: 1 }
+                    - { descr: powerSavingMode, graph: 0, value: 18, generic: 0 }
+                    - { descr: spotMode, graph: 0, value: 19, generic: 0 }
+                    - { descr: eConversion, graph: 0, value: 20, generic: 0 }
+                    - { descr: chargerSpotmode, graph: 0, value: 21, generic: 0 }
+                    - { descr: inverterSpotmode, graph: 0, value: 22, generic: 0 }
+            -
+                oid: airIRRP100UnitStatusOperateMode
```

Comment:
This will go for all of the new ones you've added but you should set:

`index: 'airIRRP100UnitStatusOperateMode.{{ $index }}'`

here.